newlib/libc/CMakeLists.txt

```diff
@@ -53,4 +53,4 @@ add_subdirectory(time)
 add_subdirectory(tinystdio)
 add_subdirectory(uchar)
 add_subdirectory(xdr)
-add_subdirectory(locale)
+add_subdirectory(tinylocale)
```

Comment:
Should this be 
```
if (_TINY_LOCALE)
  add_subdirectory(tinylocale)
else()
  add_subdirectory(locale)
endif()

Reply:
It's just like stdio -- there's no cmake support for the legacy stdio code, and I'm planning on the same thing for locale. I dug through the existing locale code and there's just not much of use for non-cygwin platforms, but it does consume a whole lot more memory (both ROM and RAM).